Jenkinsfile

```diff
@@ -1,30 +1,36 @@
-node('Slave433') {
+node('docker') {
 
-    stage 'Cleanup workspace'
-    sh 'chmod 777 -R .'
-    sh 'rm -rf *'
+    withDockerContainer(
+        image: 'camtango_db:latest',
+        args: '-u root'
+    ) {
+        stage 'Cleanup workspace'
+        sh 'chmod 777 -R .'
+        sh 'rm -rf *'
```

Comment:
Why is it necessary to do these drastic steps on the container here?

Reply:
Agree.  Probably made sense when running on the persistent Slave443, but a fresh docker container should already be empty when we start, so this seems unnecessary.